client/state/automated-transfer/selectors.js

```diff
@@ -48,3 +54,15 @@ export const getEligibility = compose(
 	getEligibilityData,
 	getAutomatedTransfer,
 );
+
+/**
+ * Checks if the site is currently transferring
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for tranfer info
+ * @returns {bool} transferring check
+ */
+export const isAutomatedTransferTransferring = ( state, siteId ) => includes(
```

Comment:
With global selectors, yes, the names would need to be unique, and that this might lead to some verbosity is expected and probably fine (though, if possible to keep meaning and uniqueness while still being brief, all the better).

To your suggested alternative, I might point out that `automatedTransfer.isTransferring( state, siteId )` is one character _longer_ than `isAutomatedTransferTransferring( state, siteId )`.

Of course, in importing into your component, you could always rename the import:

```js
import { isAutomatedTransferTransferring as isTransferring } from 'state/selectors';
```